src/sidebar/util/test/time-test.js

```diff
@@ -5,128 +5,166 @@ const time = require('../time');
-      const t = null;
+      const date = null;
       const expect = '';
-      assert.equal(time.toFuzzyString(t, mockIntl()), expect);
+      assert.equal(time.toFuzzyString(date, undefined), expect);
     });
 
-    const testFixture = function(f) {
-      return function() {
-        const t = new Date().toISOString();
-        const expect = f[1];
-        sandbox.clock.tick(f[0] * 1000);
-        assert.equal(time.toFuzzyString(t, mockIntl()), expect);
-      };
-    };
+    [
+      { now: '1970-01-01T00:00:10.000Z', text: 'Just now' },
+      { now: '1970-01-01T00:00:29.000Z', text: 'Just now' },
+      { now: '1970-01-01T00:00:49.000Z', text: '49 secs' },
+      { now: '1970-01-01T00:01:05.000Z', text: '1 min' },
+      { now: '1970-01-01T00:03:05.000Z', text: '3 mins' },
+      { now: '1970-01-01T01:00:00.000Z', text: '1 hr' },
+      { now: '1970-01-01T04:00:00.000Z', text: '4 hrs' },
+    ].forEach(test => {
+      it('creates correct fuzzy string for fixture ' + test.now, () => {
+        const timeStamp = fakeDate('1970-01-01T00:00:00.000Z');
+        const now = fakeDate(test.now);
+        assert.equal(time.toFuzzyString(timeStamp, now), test.text);
+      });
+    });
 
-    for (let i = 0, f; i < FIXTURES_TO_FUZZY_STRING.length; i++) {
-      f = FIXTURES_TO_FUZZY_STRING[i];
-      it('creates correct fuzzy string for fixture ' + i, testFixture(f));
-    }
+    [
+      {
+        now: '1970-01-02T03:00:00.000Z',
+        text: '2 Jan',
+        options: { day: 'numeric', month: 'short' },
+      },
+      {
+        now: '1970-01-04T00:30:00.000Z',
+        text: '4 Jan',
+        options: { day: 'numeric', month: 'short' },
+      },
+      {
+        now: '1970-07-03T00:00:00.000Z',
+        text: '3 July',
+        options: { day: 'numeric', month: 'short' },
+      },
+      {
+        now: '1971-01-01T00:00:00.000Z',
+        text: '1 Jan 1970',
+        options: { day: 'numeric', month: 'short', year: 'numeric' },
+      },
+      {
+        now: '1971-03-01T00:00:00.000Z',
+        text: '1 Jan 1970',
+        options: { day: 'numeric', month: 'short', year: 'numeric' },
+      },
+      {
+        now: '1972-01-01T00:00:00.000Z',
+        text: '1 Jan 1970',
+        options: { day: 'numeric', month: 'short', year: 'numeric' },
+      },
+      {
+        now: '1978-01-01T00:00:00.000Z',
+        text: '1 Jan 1970',
+        options: { day: 'numeric', month: 'short', year: 'numeric' },
+      },
+    ].forEach(test => {
+      it(
+        'passes correct arguments to format fuzzy string date for fixture ' +
+          test.now,
+        () => {
+          const timeStamp = fakeDate('1970-01-01T00:00:00.000Z');
+          const now = fakeDate(test.now);
+
+          fakeIntl.DateTimeFormat().format.returns(test.text); // eslint-disable-line new-cap
+          assert.equal(time.toFuzzyString(timeStamp, now, fakeIntl), test.text);
+          assert.calledWith(fakeIntl.DateTimeFormat, undefined, test.options);
```

Comment:
These two lines can be combined into one `calledWith` check

Reply:
How are you thinking of combining these together? I'm afraid I'm not seeing it. I could see just getting rid of the first check all-together except that we might perhaps still want to check to see that the value from the format function was actually returned.

Reply:
Please ignore this. I misread the two "calledWith" asserts as having the same first argument. I've just realized that is not the case.